plugin/pkg/admission/security/podsecurity/admission.go

```diff
@@ -178,9 +178,6 @@ var (
 )
 
 func (p *Plugin) Validate(ctx context.Context, a admission.Attributes, o admission.ObjectInterfaces) error {
-	if !p.enabled {
```

Comment:
also drop the enabled field and the `c.enabled = featureGates.Enabled(features.PodSecurity)` line